src/Utilities/StringUtil.php

```diff
@@ -21,7 +21,7 @@ final class StringUtility {
 	 *
 	 * @return bool True if the $string starts with $starts_with, false otherwise.
 	 */
-	public static function starts_with( string $string, string $starts_with, bool $case_sensitive = true ): bool {
+	public static function starts( string $string, string $starts_with, bool $case_sensitive = true ): bool {
```

Comment:
I think `starts_with` and `ends_with` are fine, also that's the name other languages use (https://docs.microsoft.com/en-us/dotnet/api/system.string.startswith, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith, https://docs.python.org/3/library/stdtypes.html#str.startswith) so maybe it's better not to deviate from "the norm" here.